admin/app/components/users/certification-center-memberships.js

```diff
@@ -1,7 +1,22 @@
 import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { inject as service } from '@ember/service';
 
 export default class CertificationCenterMemberships extends Component {
+  @service notifications;
+
   get orderedCertificationCenterMemberships() {
     return this.args.certificationCenterMemberships.sortBy('certificationCenter.name');
   }
+
+  @action
+  async disableCertificationCenterMembership(certificationCenterMembership) {
+    try {
+      certificationCenterMembership.deleteRecord();
+      await certificationCenterMembership.save();
```

Comment:
On pourrait simplifier à une seule ligne.

```suggestion
      await certificationCenterMembership.destroyRecord();
```